### ISSUE TITLE
nimsuggest-mode: Invalid face reference: face

### ISSUE THREAD

**Issue by kaushalmodi** [nim-lang/nim-mode#161]
For this simple code where I was experimenting with `noSideEffect`:

```
proc sum(x, y:int): int {. noSideEffect .} =
  echo x # causes side effect
  x + y

echo sum(2,3)
```

and with `nimsuggest-mode` enabled and `flycheck-mode` disabled, I get this in the \*Messages\* buffer:

> Invalid face reference: face

each time, I place the point on 

```
proc sum(x, y:int): int {. noSideEffect .} =
```

---

- OS: RHEL 6.6
- Emacs: version 26+, built from master branch
- Nim: v0.17.0 (verified that this issue occurs for v0.16.0 too) (git hash: d3f0f87e81e262ee25fd528aae5b6db0bdf78d5e, active boot switches: -d:release)
- nimsuggest: v0.17.0 (verified that this issue occurs for v0.16.0 too) (git hash: d3f0f87e81e262ee25fd528aae5b6db0bdf78d5e, active boot switches: -d:release --gc:markAndSweep)
- nim-mode: latest version from Melpa

---

- [x] Are you using latest version of nim-mode?
  - Yes
- [x] Are there message related to nim-mode in `*Messages*` buffer?
  - No. Just that I get *Invalid face reference: face* each time I put the point on `proc sum(x, y:int): int {. noSideEffect .} =`.
- [x] Did you read README.md?
  - Yes.
- [x] Is your problem related to nimsuggest-mode? (company-mode, flycheck, or el-doc)
   - Yes, `nimsuggest-mode`.
   - [x] Did you try nim-mode without nimsugggest-mode? What happened?
     - The issue goes away if I disabled `nimsuggest-mode`.

**Comment by kaushalmodi** [comment]
I think it has to do with the pragma.. if I remove the pragma from that first line, the invalid face error goes away:

```
proc sum(x, y:int): int {. noSideEffect .} =
```

**Comment by yuutayamada** [comment]
I'm not sure this is related to #166, but I could only reproduce on nimscript-mode and I think I fixed #166. Please re-open if this issue is still remaining.
